src/main/java/api/Controller.java

```diff
@@ -37,24 +37,45 @@ public Hello hello(@RequestParam(value = "name", defaultValue = "team3") String
     }
 
     /**
-     * An endpoint that returns a specific movie
+     * An endpoint that returns a specific movie. If two movies share the same title,
+     * the oldest is returned. To get a specific movie of a certain year, add the
+     * year paramter, i.e. /movie/titanic?year=1997
```

Comment:
Mistyped parameter in the comment here. The rest looks good!

Reply:
I saw that, in the commit message too 👎